src/_data/companies.yml

```diff
@@ -269,12 +269,12 @@ LambdaClass:
   description: Software consultancy and development company, with experience in Erlang and Elixir.
 
 Leapsight:
-  industry: Information Technology
+  industry: Information Technology, IoT
   url: http://www.leapsight.com
   blog: http://www.leapsight.com/
   github: https://gitlab.com/leapsight
-  location: London, United Kingdom; Buenos Aires, Argentina; Rosario, Argentina
-  description: We use Erlang to do scalable, interactive, intelligent systems. Erlang also powers our products - Bondy, an OSS WAMP Broker and API Gateway; and LSD, a distributed, semantic and deductive database.
+  location: London, United Kingdom; Buenos Aires, Argentina
+  description: We use Erlang to do scalable, real-time, intelligent systems. Erlang powers our products: Bondy (open source networking platform for MicroServices and IoT, a WAMP Router and REST API GAteway), LSD (distributed, knowledge graph and deductive database) and Leapsight IoT Platform.
```

Comment:
Hi @aramallo . It seems as though the `:` in the sentence is making jekyll's yaml parser unhappy. 

```
jekyll 3.8.6 | Error:  (/home/travis/build/starbelly/erlang-companies/src/_data/companies.yml): mapping values are not allowed in this context at line 277 column 104
```

Reply:
Thanks @starbelly , I will fix that